python/lsst/fgcmcal/fgcmMakeLut.py

```diff
@@ -525,10 +557,26 @@ def _fgcmMakeLut(self, camera, opticsHandle, sensorHandleDict,
                 throughputAtMax=passband[-1],
             )
 
+        standardPassbands = {}
+        for i, physical_filter in enumerate(self.fgcmLutMaker.filterNames):
+            if physical_filter != self.fgcmLutMaker.stdFilterNames[i]:
+                # This is not a standard filter.
+                continue
+            band = filterToBand[physical_filter]
+            passband = self.fgcmLutMaker.throughputs[i]['THROUGHPUT_AVG']*self.fgcmLutMaker.atmStdTrans
+            passbandTable = Table(
+                {
+                    "wavelength": (self.fgcmLutMaker.atmLambda.astype(np.float64)/10.)*units.nm,
+                    "throughput": (passband*100.)*units.percent,
+                },
+            )
+            standardPassbands[band] = passbandTable
```

Comment:
Is there anything useful you could add to the column descriptions and/or table metadata, like if the wavelength is the center of the bin or an edge, or what parts of the system the throughput includes?